### DISCUSSION TITLE
Example for update query

### DISCUSSION THREAD

**Discussion by marioscrock** [CLARIAH/grlc#498] [Q&A]
@c-martinez @GenEars I saw in the release notes that UPDATE queries are now supported. I could find only this [example](https://github.com/CLARIAH/grlc/pull/400#issuecomment-1562283086), but I cannot replicate it. Is there any example repository/query for this feature?

**Comment by GenEars** [comment]
Dear @marioscrock (sorry for the late reply!), here are query examples developed by [@MiharyR](https://github.com/MiharyR) when he was involved in https://github.com/Orange-OpenSource/grlc => [orange-grlc-test.zip](https://github.com/user-attachments/files/17930044/orange-grlc-test.zip)

Depending on where you put these queries in your query repository, you can reach them via grlc through the following paths (a GitLab repo through a local grlc instance in what follows):

- branch = main
  - folder = .
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project/branch/main/
  - folder = ./my_subdir
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project/subdir/my_subdir
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project/branch/main/subdir/my_subdir
- branch = development
  - folder = .
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project/branch/development/
  - folder = ./my_subdir
    - http://127.0.0.1:8088/api-gitlab/example-group/example-project/branch/development/subdir/my_subdir

As you suggested, it could be interesting to share these queries in some public repo for demo and testing purposes. I guess https://github.com/CLARIAH/grlc-queries/ is here for that (@c-martinez ?).

I hope the above answer helps in the meantime.

PS : @c-martinez there's currently a broken link for https://github.com/CLARIAH/grlc-queries/blob/master/endpoint-method.rq
